README.md

```diff
@@ -56,14 +56,15 @@ Encrypting your K8s is good for:
 <a href="https://landscape.cncf.io/?selected=constellation"><img src="https://raw.githubusercontent.com/cncf/artwork/1c1a10d9cc7de24235e07c8831923874331ef233/projects/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.svg" align="right" width="100px"></a>
 
 * Constellation is a [CNCF-certified][certified] Kubernetes. It's aligned to Kubernetes' [version support policy][k8s-version-support] and will likely work with your existing workloads and tools.
-* Support for Azure and GCP, more to come.
+* Support for Azure and GCP (more to come).
+* Support for local and edge installations with [MiniConstellation][first-steps-local].
```

Comment:
@datosh @m1ghtym0 Do you think it's ok to promote minicon for edge installations? (And also to compare it with K3s and MicroK8s?) It seems a bit contradictory to our reply to the bare metal issue. Also, as I understood @daniel-weisse, a future edge variant of Constellation would probably not be based on minicon, but something new on the underlying "qemu create".

Reply:
I have the same question. I have added this to our product sync agenda, since I don't have a clear answer myself, but we should find a common understanding here.

Reply:
Agree with Thomas, MiniConstellation should be used for testing only.
We should maybe just add some details for QEMU based Constellation (e.g. MiniConstellation but you perform init yourself), which we can then maybe advertise for edge deployments.
In that case, it would also be great if we can at least offer *some* support for CC in QEMU by supporting SEV-ES, and keeping track of changes to QEMU that would allow us to enable SEV-SNP

Reply:
Ok, I'll remote the edge thing...